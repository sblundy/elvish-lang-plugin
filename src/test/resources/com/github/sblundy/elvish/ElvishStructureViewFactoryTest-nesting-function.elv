fn test-method {
  fn nested-method {
    echo nested
  }
  echo ok
}