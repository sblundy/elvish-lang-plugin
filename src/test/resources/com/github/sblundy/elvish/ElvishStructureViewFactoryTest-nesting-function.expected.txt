-ElvishStructureViewFactoryTest-nesting-function.elv
 -test-method
  nested-method